azero/contracts/most/lib.rs

```diff
@@ -409,9 +426,49 @@ pub mod most {
             // ETH_ZERO_ADDRESS as `dest_token_address` indicates native ether transfer
```

Comment:
This comment can be removed (or updated). Imho `NATIVE_MARKER_ADDRESS` is self-explanatory enough
```suggestion
```